src/include/duckdb/common/types/value.hpp

```diff
@@ -12,14 +12,23 @@
 #include "duckdb/common/exception.hpp"
 #include "duckdb/common/types.hpp"
 
+
+#ifdef _WIN32
+#define DUCKDBAPI __declspec(dllexport)
```

Comment:
Can we rename this from `DUCKDBAPI` to `DUCKDB_API`, for consistency reasons?

Reply:
We can. But I thought since it was `DUCKDBAPI` in `duckdb.h`, I thought it is what I should keep everywhere. Do you want me to rename in the `duckdb.h` file as well?

Reply:
Yes, please rename it there as well.